gui/src/components/NewVersionWarning.tsx

```diff
@@ -0,0 +1,43 @@
+import CloseIcon from "@mui/icons-material/Close";
+import { Alert, Collapse, IconButton } from "@mui/material";
+import { useState } from "react";
+
+import packageJson from "../../../package.json";
+
+export async function getLatestVersion() {
+  const response = await fetch(
+    "https://api.github.com/repos/iron-wallet/iron/releases"
+  );
+  const json = await response.json();
+  return json[0].tag_name;
+}
+
+const latestVersion = await getLatestVersion();
```

Comment:
should this be done inside the component? I'm not sure if there are other implications of it being done here